ui/component/userVerify/view.jsx

```diff
@@ -44,13 +44,13 @@ class UserVerify extends React.PureComponent<Props> {
                 rewards_program: (
                   <Button button="link" label={__('Rewards Program')} href="https://lbry.com/faq/rewards" />
                 ),
+                Refresh: <Button onClick={() => fetchUser()} button="link" label={__('Refresh')} />,
+                Skip: <Button {...skipButtonProps} button="link" label={__('Skip')} />,
               }}
             >
               If you'd like to participate our %rewards_program% to earn credits, please complete one of the steps below
```

Comment:
I think you should remove the LF (end of line) after word *below *, to be translatable with the string from app-strings.json file

Reply:
Yeah, I noticed that with a few other sentences as well.  I agree that it makes string-searching hard(er).

I left it as is since there were few other similar instances, so I don't know if it was intentional or not.